.circleci/config.yml

```diff
@@ -1591,6 +1591,30 @@ workflows:
             - attach_workspace:
                 at: /
 
+  GCP-Daily-Services-Query-Restart-Performance-6N-6C:
+    triggers:
+      - schedule:
+          cron: "0 6 * * 2,6"
```

Comment:
As per last conversation in scrum, shouldn't this scheduled only on Wednesday similar to 20N_20C test ?

Reply:
I keep it as twice for now to observe it runs fine. Will change it to once per week now.

Reply:
Thanks ! Wednesday might be better instead of weekend as we can validate during the week.